quote-server/server.js

```diff
@@ -12,11 +12,31 @@ const app = express();
 //   /quotes            - Should return all quotes (json)
 //   /quotes/random     - Should return ONE quote (json)
 app.get("/", (request, response) => {
-  response.send("Neill's Quote Server!  Ask me for /quotes/random, or /quotes");
+  response.send("Areeb's Quote Server!  Ask me for /quotes/random, or /quotes");
 });
 
 //START OF YOUR CODE...
 
+app.get("/quotes", function (request, response) {
+  response.send(quotes);
+});
+
+app.get("/quotes/random", function (request, response) {
+  response.send(pickFromArray(quotes));
+});
+
+app.get("/quotes/search", function (request, response) {
+  let searchQuery = request.query.term.toLowerCase();
+  if (searchQuery) {
```

Comment:
What would happen if a client made a request to `/quotes/search` without specifying any `term`? (Not passing an empty one, just not specifying a querystring at all)? What do you expect to happen?